indigo_app/forms.py

```diff
@@ -250,6 +251,56 @@ def filter_queryset(self, queryset):
         return queryset
 
 
+class WorkFilterForm(forms.Form):
+    search_string = forms.CharField()
+    stub = forms.ChoiceField(choices=[('excl', 'excl'), ('all', 'all'), ('only', 'only')])
+    status = forms.MultipleChoiceField(choices=[('published', 'published'), ('draft', 'draft')])
+    subtype = forms.ChoiceField(choices=[('-', '-'), ('by-law', 'by-law'), ('gn', 'gn'), ('p', 'p'), ('si', 'si')])
```

Comment:
Subtypes can be loaded from the `indigo_api.Subtype` model. I think a ModelChoiceField will suffice.